docs/howto/add_relationship_links.md

```diff
@@ -0,0 +1,132 @@
+[Back to Guides](../README.md)
+
+# How to add relationship links
+
+AMS offers you many ways to add links in your JSON, depending on your needs.
```

Comment:
I think it's worth mentioning that a (maybe 'the most') common use case for links is supporting nested resources.